client/src/js/Display.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
 		this.chatboxesMap = {};
 		this.chatBox = new InputBox(
 			this.game.input,
-			new TextBox(this.term, 4, 25, { x: 0, y: 1}, this),
+			new TextBox(this.term, 4, 24, { x: 0, y: 1}, this),
```

Comment:
This is because I changed the wrapping method slightly, allowing some words to sit closer to the right-side of the box. At 25 width, there is no space between the player talk input and the game message log.